packages/flutter/test/material/time_picker_test.dart

```diff
@@ -197,4 +205,85 @@ void main() {
       expect(feedback.hapticCount, 3);
     });
   });
+
+  group('localization', () {
+    testWidgets('can localize the header for all known localizations', (WidgetTester tester) async {
+      final Map<Locale, List<String>> locales = <Locale, List<String>>{
```

Comment:
the name of the test is "all known localizations" but that's not what the test is actually testing as far as I can tell.